yt_dlp/extractor/crunchyroll.py

```diff
@@ -490,18 +490,18 @@ class CrunchyrollMusicIE(CrunchyrollBaseIE):
     _VALID_URL = r'''(?x)
         https?://(?:www\.)?crunchyroll\.com/
         (?P<lang>(?:\w{2}(?:-\w{2})?/)?)
-        watch/(?P<type>concert|musicvideo)/(?P<id>\w{10})'''
+        watch/(?P<type>concert|musicvideo)/(?P<id>\w+)'''
     _TESTS = [{
-        'url': 'https://www.crunchyroll.com/watch/musicvideo/MV88BB7F2C',
+        'url': 'https://www.crunchyroll.com/de/watch/musicvideo/MV5B02C79',
```

Comment:
add the new test instead of replacing the old one

Reply:
I have made the changes but I guess there are some issues with the linting of the code.
Is there any command to fix it ?

Reply:
If you look at the output from flake8, it will show you the line numbers and descriptions of the errors

I've added suggestions of what needs to be fixed. Though in the future this is something you should try to do before pushing the commit (run flake8 and fix problems)